inspirehep/utils/helpers.py

```diff
@@ -78,3 +82,27 @@ def force_force_list(data):
     elif isinstance(data, (tuple, set)):
         return list(data)
     return data
+
+
+def get_recid_from_url(reference):
+    """Retrieve recid from record URL.
+
+    If no recid can be parsed, return False.
+    """
+    try:
+        res = int(reference.split('/')[-1])
+    except (ValueError, AttributeError):
+        res = False
+    return res
```

Comment:
Moving this method outside of the DoJSON utils makes sense, because it's used elsewhere, but this PR is a little too early for that. I'll discard these changes.